templates/openstack/index.html

```diff
@@ -1,24 +1,83 @@
 {% extends "openstack/_base_openstack.html" %}
 
-{% block title %}OpenStack on Ubuntu is your scalable private cloud, by Canonical{% endblock %}
-{% block meta_description %}Ubuntu OpenStack is the fastest, easiest and most robust way to build your cloud - and it includes a suite of tools to help you build and manage it.{% endblock meta_description %}
+{% block title %}Canonical OpenStack on Ubuntu is the reference open source private cloud{% endblock %}
+{% block meta_description %}Ubuntu OpenStack is the most robust and economical way to build your private OpenStack cloud - with support or even full remote management by Canonical, the cloud Linux leaders.{% endblock meta_description %}
 
 {% block content %}
 <section class="p-strip is-deep is-bordered">
   <div class="row">
     <div class="col-7">
-      <h1>The number one OpenStack</h1>
+      <h1>OpenStack Delivered</h1>
+      <p>Canonical builds and manages OpenStack private clouds that match public cloud pricing and performance.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-4 p-card">
+        <h4 class="p-card__title">Cloud Build Service</h4>
+        <div class="p-card__content">
+          <p>Design and deployment of OpenStack reference architecture</p>
+          <ul>
+            <li>Hardware recommendations and sizing</li>
+            <li>Fixed-price deployment</li>
+            <li>Reference architecture with standard storage and networking options</li>
+            <li>Hyperconverged and containerised control plane</li>
+            <li>High availability</li>
+            <li>Upgrades to future versions of OpenStack on demand</li>
+          </ul>
+          <p>$80k with two week delivery on certified hardware.</p>
+          <p><a class="p-button--positive" href="/download/cloud">Get OpenStack&nbsp;&rsaquo;</a></p>
```

Comment:
the link should be /openstack/consulting (you need to look at the beta to guess the urls)